streamz/core.py

```diff
@@ -319,18 +339,35 @@ def remove_stream(change):
 
         return output._ipython_display_(**kwargs)
 
-    def _emit(self, x):
+    def _emit(self, x, metadata=None):
+        if metadata is None:
+            metadata = {}
+
+        refs = []
+        if 'refs' in metadata:
```

Comment:
Remove L343/344 in favour of `refs = metadata.get('refs', []) if metadata else []`

Reply:
This has been cleaned up. In the code we never have a list of refs, only a list of metadata.